internal/controllers/utils/utils_test.go

```diff
@@ -2088,3 +2087,116 @@ var _ = Describe("OverrideClusterInstanceLabelsOrAnnotations", func() {
 		Expect(dstClusterRequestInput).To(Equal(expected))
 	})
 })
+
+const testTemplate = `{
+  "nodeClusterName": {
+    "type": "string"
+  },
+  "oCloudSiteId": {
+    "type": "string"
+  },
+  "policyTemplateParameters": {
+    "description": "policyTemplateParameters.",
+    "properties": {
+      "sriov-network-vlan-1": {
+        "type": "string"
+      },
+      "install-plan-approval": {
+        "type": "string",
+        "default": "Automatic"
+      }
+    }
+  },
+  "clusterInstanceParameters": {
+    "description": "clusterInstanceParameters.",
+    "properties": {
+      "additionalNTPSources": {
+        "description": "AdditionalNTPSources.",
+        "items": {
+          "type": "string"
+        },
+        "type": "array"
+      }
+    }
+  },
+  "required": [
+    "nodeClusterName",
+    "oCloudSiteId",
+    "policyTemplateParameters",
+    "clusterInstanceParameters"
+  ],
+  "type": "object"
+}`
+
+func TestExtractSubSchema(t *testing.T) {
+	type args struct {
+		mainSchema []byte
+		node       string
+	}
+	tests := []struct {
+		name          string
+		args          args
+		wantSubSchema []byte
+		wantErr       bool
+	}{
+		{
+			name: "ok",
+			args: args{
+				mainSchema: []byte(testTemplate),
+				node:       "clusterInstanceParameters",
```

Comment:
`node` is confusing, which doesn't clearly represent the meaning. Could this be "subSchemaKey"?

Reply:
Fixed